R/utils_azure.R

```diff
@@ -0,0 +1,65 @@
+# Semi-automated tests of Azure Storage
+# integration live in tests/azure/.
+# These tests should not be fully automated because they
+# automatically create buckets and upload data,
+# which could put an unexpected and unfair burden on
+# external contributors from the open source community.
+# nocov start
+azure_blob_exists <- function(container,
+  blob_path,
+  version = NULL,
+  verbose = FALSE
+) {
+  tryCatch(
+    AzureStor::storage_file_exists(
+      container = container,
+      file = blob_path
+    ),
+    http_404 = function(condition) {
+      FALSE
+    }
+  )
+}
+
+azure_get_token <- function() {
+  AzureAuth::get_azure_token(Sys.getenv("RTARGETS_AZURE_APPID"),
```

Comment:
Are there standard names for these environment variables? The `RTARGETS_` prefix gives me pause. AWS and GCP use well-established environment variables documented in the services themselves or external packages. `targets` should make use of as much precedent as possible.

Reply:
I do think your intuition is correct, and there will be a more standard way of storing and accessing credentials. I have worked with GCP before, and overall their workflow is more unified than what I've found in Azure so far.

I'll comb through the Azure docs and see what I can find there.